common/lib/xmodule/xmodule/tests/test_course_module.py

```diff
@@ -142,6 +143,16 @@ def test_may_certify(self):
         self.assertFalse(self.future_noshow_certs.may_certify())
 
 
+class CourseSummaryHasEnded(unittest.TestCase):
+    """ Test for has_ended method when end date is missing timezone information. """
+
+    def test_course_end(self):
+        test_course = get_dummy_course("2012-01-01T12:00")
+        bad_end_date = parser.parse("2012-02-21 10:28:45")
+        summary = xmodule.course_module.CourseSummary(test_course.id, end=bad_end_date)
```

Comment:
I verified in the debugger that this test case does in the try/except block, and that the logged message is properly formatted.